drivers/sensor/xbr818/xbr818.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_XBR818_XBR818_H_
+#define ZEPHYR_DRIVERS_SENSOR_XBR818_XBR818_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/kernel.h>
+
+#define SENSOR_ATTR_XBR818_DELAY_TIME  (SENSOR_ATTR_PRIV_START)
+#define SENSOR_ATTR_XBR818_LOCK_TIME   (SENSOR_ATTR_PRIV_START + 1)
+#define SENSOR_ATTR_XBR818_NOISE_FLOOR (SENSOR_ATTR_PRIV_START + 2)
+#define SENSOR_ATTR_XBR818_RF_POWER    (SENSOR_ATTR_PRIV_START + 3)
```

Comment:
Those need to be in a *public* header file, which this file isn't, if you want people to be able to use this :) i.e. they need to be moved to a `include/zephyr/drivers/sensor/xbr818.h` header. 
Also please make sure this has proper Doxygen comments. See e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/drivers/sensor/tmag5273.h#L36-L49

Reply:
Done.

Reply:
> Also please make sure this has proper Doxygen comments.

Can you please take another pass, as Doxygen comments are missing (see, a good reason for not closing conversations after all 😉 ).
https://builds.zephyrproject.io/zephyr/pr/81013/docs/doxygen/html/xbr818_8h.html#a2f524848967d71fe6c7557355a064bec
you want something like tmag5273.h that I pointed you to, where the enum and all its fields have proper doxygen comments. (

Reply:
Then I have not properly identified what constitute a proper doxygen comment, what is it? the /** instead of /* at the start?

Reply:
Ok, figured it out.